src/Aspire.Hosting.Azure.ServiceBus/AzureServiceBusExtensions.cs

```diff
@@ -30,23 +30,33 @@ public static IResourceBuilder<AzureServiceBusResource> AddAzureServiceBus(this
     {
         builder.AddAzureProvisioning();
 
-        var configureInfrastructure = static (AzureResourceInfrastructure infrastructure) =>
+        var configureInfrastructure = (AzureResourceInfrastructure infrastructure) =>
```

Comment:
Can you revert this?

Reply:
Yep, with the idea to set the annotation conditionally in the extension method we can remove this.